src/EnergyPlus/WindowManager.cc

```diff
@@ -3094,6 +3096,11 @@ namespace WindowManager {
         QdotRadOutRepPerArea(SurfNum) = rad_out_per_area;
 
         QRadOutReport(SurfNum) = QdotRadOutRep(SurfNum) * TimeStepZoneSec;
+
+        // Radiation emission to air
+        QAirExtReport(SurfNum) = -SurfOutsideEmiss * sigma * (1.0 - AirSkyRadSplit(SurfNum)) * surface.ViewFactorSkyIR * (Tsout_4 - pow_4(tout));
+        QHeatEmiReport(SurfNum) = Surface(SurfNum).Area * hcout * (Tsout - tout) + QAirExtReport(SurfNum);
```

Comment:
The calculation here for `QAirExtReport` duplicates a portion of the calculation above for `rad_out_per_area`, so calculate this portion separately before line 3091 and reuse it there and here.

Reply:
`rad_out_per_area` includes radiation from the sky, ground, and air altogether. Here `QAirExtReport` only intends to radiation to air.

Reply:
Yes, but `QAirExtReport` is part of `rad_out_per_area`. We don't want to do calculations more than necessary, especially `pow_4`.

Reply:
Here's the current code:
```
        Real64 const rad_out_per_area(
            -SurfOutsideEmiss * sigma *
            ((((1.0 - AirSkyRadSplit(SurfNum)) * surface.ViewFactorSkyIR + surface.ViewFactorGroundIR) * (Tsout_4 - pow_4(tout))) +
             (AirSkyRadSplit(SurfNum) * surface.ViewFactorSkyIR * (Tsout_4 - pow_4(SkyTempKelvin))) + QRadLWOutSrdSurfs(SurfNum)));

        QAirExtReport(SurfNum) = -SurfOutsideEmiss * sigma * (1.0 - AirSkyRadSplit(SurfNum)) * surface.ViewFactorSkyIR * (Tsout_4 - pow_4(tout));
```
This can be split apart to avoid the duplicate calculation:
```
        Real64 const rad_out_air_per_area = -SurfOutsideEmiss * sigma * (1.0 - AirSkyRadSplit(SurfNum)) * surface.ViewFactorSkyIR * (Tsout_4 - pow_4(tout));
        Real64 const rad_out_per_area = rad_out_air_per_area + (-SurfOutsideEmiss * sigma *
            ((AirSkyRadSplit(SurfNum) * surface.ViewFactorSkyIR * (Tsout_4 - pow_4(SkyTempKelvin))) + QRadLWOutSrdSurfs(SurfNum)));

        QAirExtReport(SurfNum) = surface.Area * rad_out_air_per_area;
        QHeatEmiReport(SurfNum) = surface.Area * hcout * (Tsout - tout) + QAirExtReport(SurfNum);
```
I think QAirExtReport was missing the surface area before?
Also, `QHeatEmiReport` should use `surfface.Area` instead of `Surface(SurfNum).Area`.